packages/legend-studio/src/stores/editor-state/element-editor-state/RuntimeEditorState.ts

```diff
@@ -200,13 +202,33 @@ export const isConnectionForModelStoreWithClass = (
   return false;
 };
 
+export const isConnectionForModelStoreWithMapping = (
```

Comment:
Why can't we merge this with `isConnectionForModelStoreWithClass`?

Reply:
will do that